### PR TITLE
DOCS Update: Using Stylesheets.tid

### PR DESCRIPTION
**Pull request by 00SS**
See Issue #3726

### CODE REVIEW

**Comment by 00SS** [comment]
You can see how it looks here: https://00ss.github.io/documentation/tiddlywiki.html#Using%20Stylesheets

File: editions/tw5.com/tiddlers/howtos/Using Stylesheets.tid [hunk 1]
```diff
@@ -55,9 +72,9 @@ body.tc-body pre {
...
+The `\rules` [[pragma|Pragma]] at the top of the tiddler restricts the ~WikiText to just allow macros and transclusion. This avoids mistakenly triggering unwanted ~WikiText processing.
 
-A stylesheet tiddler is processed such that it is first wikified and then the text portion of the ouput is extracted to apply as the CSS. Any HTML tags you will use in your stylesheet are thus ignored. For example, HTML elements generated by the RevealWidget will not affect the output. As in the following example, you can wrap CSS rules in `<pre>` tags to display them as a codeblock without affecting processing, including handling the inner macro.
+The stylesheet tiddler is processed such that it is first Wikified and then the text portion of the ouput is extracted to apply as the CSS. Any HTML tags you will use in your stylesheet are thus ignored. For example, HTML elements generated by the RevealWidget will not affect the output. As in the following example, you can wrap CSS rules in `<pre>` tags to display them as a codeblock without affecting processing, including handling the inner macro.
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/howtos/Using Stylesheets.tid]: Hi @00SS I think this might be accidental capitalisation?

**Comment by twMat** [comment]
https://00ss.github.io/documentation/tiddlywiki.html#Using%20Stylesheets

Please note that in the button labeled ControlPanel, the CamelCase autolinking interferes with the button.

**Comment by 00SS** [comment]
Made the two corrections:
1. Wikified --> wikified
1. CamelCase of ControlPanel fixed

File: editions/tw5.com/tiddlers/howtos/Using Stylesheets.tid (hunk 1 above)
Comment (00SS) [reply editions/tw5.com/tiddlers/howtos/Using Stylesheets.tid]: Thanks! Fixed!

**Pull request closed without merging**